tools/builder/sass.js

```diff
@@ -60,6 +60,7 @@ gulp.task( 'sass:dops', function( done ) {
 
 function doRTL( files, done ) {
 	let dest = './inc/build',
```

Comment:
Not sure when this was changed, but the correct path should be `./_inc/` - getting a new directory after building fresh.

Reply:
yup, that was me in https://github.com/Automattic/jetpack/commit/aed1c420514da9474d3587d1d8930d850d4bafa6 -- fixed in 6edec5d